tests/client/test_data_sender.py

```diff
@@ -35,3 +35,21 @@ async def test_post_events(
         response = await gundi_data_sender_client_v2.post_events([event_payload])
         assert response == events_created_response
         assert events_api_mock.called
+
+
+@pytest.mark.asyncio
+async def test_post_event_attachment(
+    gundi_data_sender_client_v2, event_attachment_payload
+):
+    async with respx.mock(assert_all_called=False) as gundi_api_mock:
+        # Mock API response
+        event_id = "dummy-123"
+        events_endpoint = f"{gundi_data_sender_client_v2.sensors_api_endpoint}/events/{event_id}/attachments/"
+        events_api_mock = gundi_api_mock.post(events_endpoint).respond(
+            status_code=httpx.codes.OK,
+            json={}
```

Comment:
Thank you for adding tests!. If you want a more realistic response in the mock you can use something like this here:
```JSON
{
    "object_id": "af8e2946-bad6-4d02-8a26-99dde34bd9fa",
    "created_at": "2024-07-04T13:15:26.559894Z",
    "updated_at": null
}
```